deploy/olm-catalog/apicast-community-operator/apicast-community-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
     capabilities: Full Lifecycle
     categories: "Integration & Delivery"
     certified: "false"
-    containerImage: quay.io/3scale/apicast-operator:master
+    containerImage: quay.io/3scale/apicast-operator:v0.1.0
```

Comment:
Assume first version (and image) of the operator will be v0.1.0. Is that ok for you?

Reply:
see comments above